examples/figshare_example.py

```diff
@@ -0,0 +1,36 @@
+import sys
+from distutils.command.config import config
+
+import yaml
+from  dotenv import load_dotenv
+import os
+from pprint import *
+
+import fairly
+
+
+load_dotenv('./.env')
+
+if __name__ == "__main__":
+    # Argv is the token to use
+    if len(sys.argv) > 1:
+        token = sys.argv[1]
+    else:
+        token = os.environ['FOURTU_TOKEN']
```

Comment:
There is not need to use load_dotenv(), the package automatically loads related information (e.g. token) from the configuration file and environmental variables. Giving such an example creates as alternative approach, which differs from the native approach of the package.

Reply:
I think this is a point that should be treated in the documentation. And keep the user's documentation apart from the developer's documentation. For example, this example might be useful for developers but not for users. So, maybe we should distinguish between dev-examples, and user-examples.